src/gc.c

```diff
@@ -600,7 +600,7 @@ void gc_unroot_roots(GarbageCollector* gc)
 size_t gc_stop(GarbageCollector* gc)
 {
     gc_unroot_roots(gc);
-    size_t collected = gc_sweep(gc);
+    size_t collected = gc_sweep(gc, true);
```

Comment:
Either `#define` or use an `extern const` to name the second parameter. It's not clear from the call what should be happening (see my general comments).